marketplacetests/firefox_accounts/app.py

```diff
@@ -50,5 +71,20 @@ def tap_next(self):
 
     def tap_sign_in(self):
         self.keyboard.dismiss()
+        self.change_to_correct_login_frame()
```

Comment:
I agree with this, it would be an improvement.
I tried saving the current frame after we switch it and returning that. But it switches me to the app that triggered the log in and not the log in frame :|
Do you have any suggestions on how to implement this? Thanks!